src/Icons.tsx

```diff
@@ -87,3 +87,27 @@ export const ForkIcon = makeIcon(
     viewBox: '0 0 512 512',
   },
 );
+
+export const Lightbulb = makeIcon(
```

Comment:
are those icons taken from any external source? would be great to link to the source here, for each icon. Are there any licensing concerns related to using those icons?

Reply:
Yes, Material! No licensing concerns

Reply:
https://github.com/google/material-design-icons/blob/master/LICENSE

Reply:
According to this text, we should include a copy of their LICENSE somewhere - but quite frankly, I never know how to do it properly.